file_upload_cli.py

```diff
@@ -4,7 +4,7 @@
 import requests
 
 
-def upload_files(directory_path, agent_id):
+def upload_files(repo, directory_path, agent_id):
```

Comment:
Is a 'repo' an arbitrary name given to track a "set" of documents?

Reply:
yes, more info in comment below.